src/theory/strings/sequences_rewriter.cpp

```diff
@@ -1527,7 +1526,7 @@ Node SequencesRewriter::rewriteSeqNth(Node node)
     if (pos < len)
     {
       std::vector<Node> elements = s.getConst<Sequence>().getVec();
-      ret = elements[pos];
+      Node ret = elements[pos];
```

Comment:
To avoid the copy? :-)
```suggestion
      const Node& ret = elements[pos];
```